### PR TITLE
[NEW] Check fiscal_category_id.journal_id and lines.fiscal_category_i…

### PR DESCRIPTION
**Pull request by mileo**
…d.journal_id with invoice.journal_id

### CODE REVIEW

File: l10n_br_account/models/account_invoice.py [hunk 1]
```diff
@@ -126,6 +126,22 @@ def _default_fiscal_document_serie(self):
     _order = 'internal_number desc'
 
     @api.one
+    @api.constrains('fiscal_category_id', 'jornal_id', 'invoice_line')
+    def _check_fiscal_category_journal(self):
+        if (self.fiscal_category_id and
+            self.fiscal_category_id.journal_id != self.journal_id):
```
Comment (renatonlima) [inline l10n_br_account/models/account_invoice.py]: deve ser: self.fiscal_category_id.property_journal

File: l10n_br_account/models/account_invoice.py [hunk 2]
```diff
@@ -126,6 +126,22 @@ def _default_fiscal_document_serie(self):
...
+            raise UserError(
+                        u'Não é possível registrar documentos fiscais com '
+                        u'categorias fiscais que tenham '
+                        u'diários diferentes do da fatura.')
+        for item in self.invoice_line:
+            if item.fiscal_category_id.journal_id != self.journal_id:
```
Comment (renatonlima) [inline l10n_br_account/models/account_invoice.py]: deve ser: self.fiscal_category_id.property_journal

**Comment by renatonlima** [comment]
:+1: A Ideia é boa, porém tem que corrigir o PEP8 e o nome do campo journal na categoria fiscal.

**Comment by mileo** [comment]
Fixed

**Comment by renatonlima** [comment]
@mileo Please check travis, there are erros in tests and PEP8

**Pull request closed without merging**